src/postgraphql/cli.ts

```diff
@@ -37,6 +37,7 @@ program
   .option('-a, --classic-ids', 'use classic global id field name. required to support Relay 1')
   .option('-j, --dynamic-json', 'enable dynamic JSON in GraphQL inputs and outputs. uses stringified JSON by default')
   .option('-M, --disable-default-mutations', 'disable default mutations, mutation will only be possible through Postgres functions')
+  .option('-y, --primary-key-api', 'enable additional endpoints to primary-key apis')
```

Comment:
Since this should probably default true, adding a command line option to disable it instead would be wise.